File: JavaSource/src/main/java/ru/akhcheck/patterns/chainresponsibility/handlers/StderrHandler.java
package ru.akhcheck.patterns.chainresponsibility.handlers;

import java.util.Set;

/** Печатает запись в поток ошибок. */
public class StderrHandler extends BaseHandler {
    public StderrHandler(Set<LogLevel> logLevels) {
        super(logLevels);
    }

    @Override
    protected void process(Log log) {
        System.err.println(log);
    }
}

File: ChainResponsibility/python-source/handlers/StderrHandler.py
from __future__ import annotations

import sys

from .BaseHandler import BaseHandler
from .Log import Log


class StderrHandler(BaseHandler):
    """Печатает запись в поток ошибок."""

    def process(self, log: Log) -> None:
        print(str(log), file=sys.stderr)
